resources/views/products.blade.php

```diff
@@ -71,14 +100,16 @@
                 </td>
                 <td>
                     <form action="{{route('productsDestroy', $product->id)}}" method="post">
-                        <input class="btn btn btn-outline-danger" type="submit" value="Delete" />
+                        <input class="btn btn btn-outline-danger" type="submit" value="Delete"/>
                         @method('delete')
                         @csrf
                     </form>
                 </td>
             </tr>
         @endforeach
-
         </tbody>
     </table>
+    <div class="d-flex">
```

Comment:
Move this pagination to partials folder. And after that `@include('partials.pagination', $products->links()')`

Reply:
Fixed